packages/mobile/e2e/README.md

```diff
@@ -176,6 +176,7 @@ If tests are failing for unknown reasons:
 - Rebuild, re-yarn and rerun. Sometimes the problem just goes away.
 - Delete snapshots in the emulator
 - Look at the emulator while the tests are running. Can you see anything obvious going wrong?
+- If running into issues building for iOS related to nvm try prefixing the build commands in `packages/mobile/package.json` with `unset PREFIX && ...`
```

Comment:
would be cool to mention why this might help here?

Reply:
Will create a new PR with some doc updates for running in the app in release mode and a more descriptive tidbit about when this change might be needed locally.